### PR TITLE
fix(lang): Add missing strings for Chinese (Simplified) and Chinese (Traditional)

### PR DESCRIPTION
**Pull request by misteroneill**
## Description
Add four missing strings for both Chinese variants we offer.

There are also script updates.

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,9 +38,11 @@
     "postbuild:css:cdn": "postcss --verbose --config postcss.config.js -d dist/alt dist/alt/video-js-cdn.css",
     "build:css:default": "sass --no-source-map src/css/vjs.scss dist/video-js.css",
     "postbuild:css:default": "postcss --verbose --config postcss.config.js -d dist/ dist/video-js.css",
+    "prebuild:lang": "npm-run-all -s prebuild:lang:*",
+    "prebuild:lang:zh-rm": "shx rm -f lang/zh-Han*.json",
```
Comment (brandonocasey) [inline package.json]: this should probably go in `clean`

File: package.json [hunk 2]
```diff
@@ -38,9 +38,11 @@
...
+    "prebuild:lang:zh-s": "shx cp lang/zh-CN.json lang/zh-Hans.json",
```
Comment (brandonocasey) [inline package.json]: I think this can stay as `build:lang:chinese-s` and `build:lang:chinese-t`

File: package.json [hunk 3]
```diff
@@ -181,7 +183,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm run prebuild:lang && git add lang/ && lint-staged"
```
Comment (brandonocasey) [inline package.json]: We should probably change `lint-staged` here rather than the husky hook. Should be a copy and past from https://github.com/videojs/video.js/pull/5649/files#r304555387

**Comment by gkatsev** [comment]
FYI, we don't have a `lang:` prefix: https://github.com/videojs/conventional-changelog-videojs/blob/master/convention.md#type

File: package.json (hunk 3 above)
Comment (gkatsev) [inline package.json]: Why is this necessary?

Comment (misteroneill) [reply package.json]: I tried but it didn't want to work. Will try again.

Comment (misteroneill) [reply package.json]: I guess it's not! I thought I was adding the clones to the repo.

**Comment by misteroneill** [comment]
Which makes more sense:

1. Continue ignoring `zh-Hans.json` and `zh-Hant.json` and only using them as temporary build files.
2. Add them to the repository, requiring more scripting to make sure they stay in sync and are added on commit.

My vote is 1.

**Comment by brandonocasey** [comment]
I prefer #1 as well
